serendipity_event_dejure/serendipity_event_dejure.php

```diff
@@ -213,9 +241,10 @@ function djo_vernetzen_ueber_cache(&$ausgangstext) {
         global $serendipity;
 
     	$schluessel = md5($ausgangstext);
-    	$rueckgabe = serendipity_db_query("SELECT cache FROM {$serendipity['dbPrefix']}dejure WHERE ckey = '" . $schluessel . "'", true, 'assoc');
+        $rueckgabe = serendipity_db_query("SELECT cache FROM {$serendipity['dbPrefix']}dejure
+                                           WHERE ckey = '".$schluessel."' AND last_update > " . (time()-86400*CACHE_VORHALT), true, 'assoc');
     	if (!empty($rueckgabe['cache']) && $rueckgabe['cache'] == "<!-- idem -->") {
-		return $ausgangstext;
+		    return $ausgangstext;
```

Comment:
Nothing that needs to be fixed right now, but it is not so nice the code uses german variable names.